be/src/serde/column_array_serde.cpp

```diff
@@ -53,41 +58,145 @@ const uint8_t* read_raw(const uint8_t* buff, void* target, size_t size) {
     return buff + size;
 }
 
-template <typename T>
+template <bool sorted_32ints>
+uint8_t* encode_integers(const void* data, size_t size, uint8_t* buff, int encode_level) {
```

Comment:
maybe add a benchmark in bench to compare with old codes?

Reply:
there are some comprehensive benchmarks to compre various encoding/compression algorithms, https://github.com/powturbo/TurboPFor-Integer-Compression/issues/47,  so we can refer to it.